integration-tests/tests/billing-projects.js

```diff
@@ -273,10 +273,19 @@ const testBillingSpendReportFn = withUserToken(async ({ page, testUrl, token })
   // Check that the Spend report tab is not visible on this page
   await billingPage.assertTextNotFound('Spend report')
 
-  // Select an Azure billing project and check that neither the Spend report tab nor the spend report configuration is visible
+  // Select an Azure billing project and check that the Spend Report tab is accessible but displaying only total cost information
   await billingPage.visit()
   await billingPage.selectProject(azureBillingProjectName, AZURE)
-  await billingPage.assertTextNotFound('Spend report')
```

Comment:
Can you add a check to `verifyAccessibility` here? It should currently fail because the link text does not have sufficient color contrast.

![image](https://user-images.githubusercontent.com/484484/227343341-a86c3854-61b4-42a1-8cb6-4dcaee039990.png)

Reply:
I added `await verifyAccessibility(page)` here, but the tests are still passing. Any ideas?

Also, does the `verifyAccessibility` at the end of the test test the same thing since there's only assertions between here and the end?

Reply:
Ugh, that's disappointing… I'll take a look, but don't worry about it. And I didn't see that there was already a call at the end of the test (it wasn't showing in the diff). So no need to add another one!

Reply:
The color-contrast check was showing as "incomplete" instead of failed. I updated the axe-puppeteer library (because others had reported this problem) and then the test failed for me. I pushed up the library update, but it's possible that this will cause other tests to fail as well.